### ISSUE TITLE
Feedback API rate limit

### ISSUE THREAD

**Issue by svenseeberg** [Integreat/cms-django#498]
Add a rate limit (per IP address) for posting feedback to the feedback API. As we do not have any authentication, we have no other methods for preventing huge amounts of spam (which is still very unlikely).

**Comment by timoludwig** [comment]
Maybe we can use similar measures like for issue #390 (General rate limiting for the API).

**Comment by mckinly** [comment]
@timoludwig I could work on this. Do you have a rate limit in mind such as `@ratelimit(key='ip', rate='100/h')`?

**Comment by svenseeberg** [comment]
Yes, 100/h seems legit. I think it would be useful to have this as a setting in the `settings.py`.

**Comment by timoludwig** [comment]
@mckinly Are you still interested in solving the issue? If not, please let us know so that others can take up the issue. Thanks!

**Comment by mckinly** [comment]
Not at this time. Sorry, guys! Maybe in the future. I unassigned myself for now

**Comment by ulliholtgrave** [comment]
This will be done by our NGINX/Apache2
